### PR TITLE
New Cool features for NativeUI (for now in FiveM but portable in Master)

### PR DESCRIPTION
**Pull request by manups4e**
- Updated everything to Master Branch methods and classes,
- Added new cool colors in Colors.cs
- Added Panels to NativeUI
- Added Heritage Window to NativeUI
- Added new Items ( UIMenuSeparatorItem and UIMenuSliderHeritageItem)
- Replaced Function.Call with actual Natives as this fasten up execution on FiveM ( https://i.imgur.com/bRh2Eim.png)
- Changed some drawing methods to async (fivem seem to like it more since it's threaded this will fasten up execution and mantain milliseconds counter at a low rate)
- Added new Methods (like the new UpdateDescription that makes you update items description without having to close and reopen your menu to see the effects)
- Changed some Point to PointF and Size to SizeF (needed for Panels and Window precision) this is backward compatible as you can use a Point instead of a PointF but not the contrary

### CODE REVIEW

**Comment by manups4e** [comment]
appveyor is configured for Master branch.. it will not work i think for fivem branch right now

File: MenuExample/MenuExample.csproj [hunk 1]
```diff
@@ -31,11 +31,12 @@
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="CitizenFX.Core, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\CitizenFX.Client.1.3.0.1130210\lib\net452\CitizenFX.Core.dll</HintPath>
+    <Reference Include="CitizenFX.Core.Client, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\CitizenFX.Core.Client.1.0.1580\lib\net45\CitizenFX.Core.Client.dll</HintPath>
```
Comment (manups4e) [inline MenuExample/MenuExample.csproj]: i've always used official fivem source libs, as they are updated constantly and afaik have more optimization features

**Comment by justalemon** [comment]
Hi, I like what this PR introduced but there is a slight problem: Everything is mixed up.

Can you please create a Pull Request per feature, fix or update? So we can approve and deny individual changes.

**Comment by manups4e** [comment]
Yup all right I'll do it asap

**Comment by justalemon** [comment]
Also: new features should be pushed onto master and then backported to FiveM.

**Comment by manups4e** [comment]
I'm not using master and cannot test on it.. if I can't pr directly on fivem is useless to pr directly  😅

**Comment by justalemon** [comment]
You need to use master sadly, because the SHVDN version is more used than FiveM.

**Comment by manups4e** [comment]
yeah i get it.. but i can't test the changes on master.. i only use FiveM.. i con only PR to FiveM branch.. it can't be ported to master directly? i mean.. even if in the same project.. FiveM branch and Master branch are 2 separate environments and could actually act on their own

**Pull request closed without merging**